cypress/integration/tests/developer/controls/businessservices/create.test.ts

```diff
@@ -124,7 +124,7 @@ describe("Business service validations", { tags: "@tier2" }, () => {
         // Check name duplication
         inputText(businessServiceNameInput, businessService.name);
         cy.get(commonView.submitButton).should("be.disabled");
-        cy.get(commonView.nameHelper).should("contain.text", duplicateName);
+        cy.get(commonView.nameHelper).should("contain.text", duplicateApplication);
```

Comment:
This will be duplicateBusinessService or duplicateApplication ? 

I don't see duplicateBusinessService being used anywhere